data/hooks/conf_regen/06-slapd

```diff
@@ -36,20 +35,31 @@ EOF
 
   DEBIAN_FRONTEND=noninteractive dpkg-reconfigure slapd -u
 
-  # Regen conf
-
-  _regenerate_slapd_conf
-
-   # Enforce permissions
-  chown root:openldap /etc/ldap/slapd.ldif
+  # Enforce permissions
   chown -R openldap:openldap /etc/ldap/schema/
   usermod -aG ssl-cert openldap
 
-  systemctl restart slapd
+  # (Re-)init data according to default ldap entries
+  echo '  Initializing LDAP with Yunohost DB structure'
 
-  # (Re-)init data according to ldap_scheme.yaml
+  rm -rf /etc/ldap/slapd.d
+  mkdir -p /etc/ldap/slapd.d
+  slapadd -F /etc/ldap/slapd.d -b cn=config -l "$config" 2>&1 \
+      | grep -v "none elapsed\|Closing DB" || true
+  chown -R openldap: /etc/ldap/slapd.d
+
+  rm -rf /var/lib/ldap
+  mkdir -p /var/lib/ldap
+  slapadd -F /etc/ldap/slapd.d -b dc=yunohost,dc=org -l "$db_init" 2>&1 \
+      | grep -v "none elapsed\|Closing DB" || true
+  chown -R openldap: /var/lib/ldap
+
+  nscd -i groups
+  nscd -i passwd
 
-  yunohost tools shell -c "from yunohost.tools import tools_ldapinit; tools_ldapinit()"
+  systemctl restart slapd
+
+  mkhomedir_helper admin
```

Comment:
Not sure this one work inside chroot cause admin user doesn't exists as slapd is down.